### DISCUSSION TITLE
user-guide/quick-start

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#119] [文档评论区]
# user-guide/quick-start

本节是 quick start guide [Zotero Documentation] (https://www.zotero.org/support/quickstartguide) 的翻译。您也可以参阅其他内部章节以获得更多详细步骤指导。 基础知识 如何安装 Zotero? 阅读 安装 Zotero (install.md) 如何打开 Zoter...

https://zotero-chinese.github.io/user-guide/quick-start.html

**Comment by yaomiaomiao** [comment]
无法获取pdf，中文英文都是，显示无法获取该条目，翻译器已经更新，浏览器的也相应更新，就是抓取失败，请问怎么处理，只能手动导入嘛？
